### ISSUE TITLE
Import error when trying to import MuonGun (Trac #2142)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix2#2134]
```text

In [1]: from icecube import MuonGun
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-1-4e878b76dfa2> in <module>()
----> 1 from icecube import MuonGun

/cvmfs/icecube.opensciencegrid.org/py2-v3/RHEL_7_x86_64/metaprojects/combo/stable/lib/icecube/MuonGun/__init__.py in <module>()
      1 from icecube.load_pybindings import load_pybindings
      2 import icecube.icetray, icecube.dataclasses, icecube.simclasses, icecube.phys_services# be nice and pull in our dependencies
----> 3 load_pybindings(__name__,__path__)
      4 
      5 import inspect

/cvmfs/icecube.opensciencegrid.org/py2-v3/RHEL_7_x86_64/metaprojects/combo/stable/lib/icecube/load_pybindings.py in load_pybindings(name, path)
     56     import imp, sys
     57     thismod = sys.modules[name]
---> 58     m = imp.load_dynamic(name, path[0] + ".so")
     59     sys.modules[name] = thismod # Some python versions overwrite the Python
     60                                 # module entry with the C++ one. We don't want

ImportError: /cvmfs/icecube.opensciencegrid.org/py2-v3/RHEL_7_x86_64/metaprojects/combo/stable/lib/icecube/MuonGun.so: undefined symbol: _ZN5boost6python5numpy10initializeEb

In [2]:

```

Migrated from https://code.icecube.wisc.edu/ticket/2142
```json
{
    "status": "closed", 
    "changetime": "2018-05-15T18:03:40", 
    "description": "{{{\n\nIn [1]: from icecube import MuonGun\n---------------------------------------------------------------------------\nImportError                               Traceback (most recent call last)\n<ipython-input-1-4e878b76dfa2> in <module>()\n----> 1 from icecube import MuonGun\n\n/cvmfs/icecube.opensciencegrid.org/py2-v3/RHEL_7_x86_64/metaprojects/combo/stable/lib/icecube/MuonGun/__init__.py in <module>()\n      1 from icecube.load_pybindings import load_pybindings\n      2 import icecube.icetray, icecube.dataclasses, icecube.simclasses, icecube.phys_services# be nice and pull in our dependencies\n----> 3 load_pybindings(__name__,__path__)\n      4 \n      5 import inspect\n\n/cvmfs/icecube.opensciencegrid.org/py2-v3/RHEL_7_x86_64/metaprojects/combo/stable/lib/icecube/load_pybindings.py in load_pybindings(name, path)\n     56     import imp, sys\n     57     thismod = sys.modules[name]\n---> 58     m = imp.load_dynamic(name, path[0] + \".so\")\n     59     sys.modules[name] = thismod # Some python versions overwrite the Python\n     60                                 # module entry with the C++ one. We don't want\n\nImportError: /cvmfs/icecube.opensciencegrid.org/py2-v3/RHEL_7_x86_64/metaprojects/combo/stable/lib/icecube/MuonGun.so: undefined symbol: _ZN5boost6python5numpy10initializeEb\n\nIn [2]:\n\n}}}", 
    "reporter": "thomas.kittler", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1526407420536099", 
    "component": "cvmfs", 
    "summary": "Import error when trying to import MuonGun", 
    "priority": "blocker", 
    "keywords": "", 
    "time": "2018-03-12T17:04:39", 
    "milestone": "", 
    "owner": "david.schultz", 
    "type": "defect"
}
```

**Comment by icecube-trac** [comment]
Trac update at 20180316T14:11:28: 
* jvansanten commented:

> This is a bug in py2-v3. Boost 1.63 contains its own implementation of boost::python::numpy, with an associated libboost_numpy. Martin's boost::numpy fork uses a different namespace (boost::numpy), but the same library name. If boost::numpy was installed after boost (as it would have to be), then it overwrote boost's library.
> 
>  


* jvansanten changed component from "simulation" to "cvmfs"

**Comment by icecube-trac** [comment]
Trac update at 20180515T17:56:23: 
* olivas changed owner from "" to "david.schultz"
* olivas changed status from "new" to "assigned"

**Comment by icecube-trac** [comment]
Trac update at 20180515T18:03:40: 
* david.schultz changed resolution from "" to "fixed"
* david.schultz commented:

> Fixed in py2-v3.0.1


* david.schultz changed status from "assigned" to "closed"
